### PR TITLE
Added community section with libs and presentation pages

### LINKED ISSUE #66: Add "Community" page to promote community libraries

### PR DESCRIPTION
**Pull request by redonkulus**
@jedireza @Vijar 

Fixes #66

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by Vijar** [comment]
:ship:

File: docs/community/libraries.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Libraries
+
+Take a look at some of the libraries that our community has built. Please use the link above to add your own library.
```
Comment (jedireza) [inline docs/community/libraries.md]: ~~Please use the link above to add your own library.~~

Edit this page on GitHub to list your libraries or plugins.

**Comment by jedireza** [comment]
Also I ran it locally and everything seems to be working as expected.

LGTM I left an alternative sentence.

:shipit: on a :ship:

File: docs/community/libraries.md (hunk 1 above)
Comment (redonkulus) [inline docs/community/libraries.md]: I like that better. Done

**Pull request merged**